### DISCUSSION TITLE
Errors running examples and tests

### DISCUSSION THREAD

**Discussion by cpittman-rsi** [vuejs/petite-vue#205] [Q&A]
I'm trying to run some of the examples, but I'm getting the error "Uncaught RangeError: Maximum call stack size exceeded" repeatedly.  I've cloned, npm installed, and npm run dev.  Some of the examples and tests work, but most throw this error related to reactivity.

To recreate, try opening the reactive test.  Clicking the global increment button works fine, but clicking the local increment button throws this error.  Or try opening the grid example, searching or sorting both trigger this error.

Any suggestions on how I can get this working?

**Comment by sqllyw** [comment]
I got the same issue

**Comment by sqllyw** [comment]
found a fix, update the packages.json:

 "@vue/reactivity": "3.2.27",

from 

 "@vue/reactivity": "^3.2.27",


the error maximum  call stack size goes away
